src/shipit_uplift/shipit_uplift/api.py

```diff
@@ -21,7 +21,7 @@
     serialize_patch_status
 )
 from shipit_uplift.config import SCOPES_USER, SCOPES_BOT, SCOPES_ADMIN
-from shipit_uplift import coverage_by_dir_impl, coverage_by_changeset_impl
+from shipit_uplift import coverage_by_dir_impl, coverage_by_changeset_impl, coverage_summary_by_changeset_impl
```

Comment:
Could you please use the following style?

```python
from x import (
    a,
    b
)

```

Reply:
Done!